metamodel-export/src/main/kotlin/org/modelix/metamodel/export/MPSMetaModelExporter.kt

```diff
@@ -0,0 +1,480 @@
+            -0x38d25d468331bbb9L,
+            -0x7c760bf823eea749L,
+            0xf979bd086aL,
+            0xf98051c244L,
+            "specializedLink",
+        )
+
+        /*package*/
+        val `target$m40F`: SReferenceLink = MetaAdapterFactory.getReferenceLink(
+            -0x38d25d468331bbb9L,
+            -0x7c760bf823eea749L,
+            0xf979bd086aL,
+            0xf98055fef0L,
+            "target",
+        )
+
+        /*package*/
+        val `members$wmsL`: SContainmentLink = MetaAdapterFactory.getContainmentLink(
+            -0x38d25d468331bbb9L,
+            -0x7c760bf823eea749L,
+            0x2e770ca32c607c5fL,
+            0x2e770ca32c607cc1L,
+            "members",
+        )
+
+        /*package*/
+        val `defaultMember$vlmG`: SReferenceLink = MetaAdapterFactory.getReferenceLink(
+            -0x38d25d468331bbb9L,
+            -0x7c760bf823eea749L,
+            0x2e770ca32c607c5fL,
+            0xeeb344f63fe016cL,
+            "defaultMember",
+        )
+    }
+
+    private object PROPS {
+        /*package*/
+        val `name$MnvL`: SProperty = MetaAdapterFactory.getProperty(
+            -0x3154ae6ada15b0deL,
+            -0x646defc46a3573f4L,
+            0x110396eaaa4L,
+            0x110396ec041L,
+            "name",
+        )
+
+        /*package*/
+        val `metaClass$PeKc`: SProperty = MetaAdapterFactory.getProperty(
+            -0x38d25d468331bbb9L,
+            -0x7c760bf823eea749L,
+            0xf979bd086aL,
+            0xf980556927L,
+            "metaClass",
+        )
+
+        /*package*/
+        val `abstract$ibpT`: SProperty = MetaAdapterFactory.getProperty(
+            -0x38d25d468331bbb9L,
+            -0x7c760bf823eea749L,
+            0x1103553c5ffL,
+            0x403a32c5772c7ec2L,
+            "abstract",
+        )
+
+        /*package*/
+        val `conceptAlias$OL_L`: SProperty = MetaAdapterFactory.getProperty(
+            -0x38d25d468331bbb9L,
+            -0x7c760bf823eea749L,
+            0x1103553c5ffL,
+            0x46ab0ad5826c74caL,
+            "conceptAlias",
+        )
+
+        /*package*/
+        val `presentation$BjyV`: SProperty = MetaAdapterFactory.getProperty(
+            -0x38d25d468331bbb9L,
+            -0x7c760bf823eea749L,
+            0x2e770ca32c607c60L,
+            0x9538e3a78561888L,
+            "presentation",
+        )
+
+        /*package*/
+        val `memberId$LVXV`: SProperty = MetaAdapterFactory.getProperty(
+            -0x38d25d468331bbb9L,
+            -0x7c760bf823eea749L,
+            0x2e770ca32c607c60L,
+            0x13b8f6fdce540e38L,
+            "memberId",
+        )
+    }
+
+    companion object {
+        private fun <T> as_y4kbwa_a0a3a8(o: Any, type: Class<T>): T? {
+            return (if (type.isInstance(o)) o as T else null)
+        }
+
+        private fun <T> as_y4kbwa_a0a0a0a0a0f0a0a0a6a01(o: Any, type: Class<T>): T? {
+            return (if (type.isInstance(o)) o as T else null)
+        }
+    }
```

Comment:
Seems like the generation has created some unreadable code. Can we refactor this? These extra methods are unneccessary overhead.

Reply:
I did some additional cleanup of the converted code.